pkg/url/url.go

```diff
@@ -8,32 +8,52 @@ import (
 	applabels "github.com/redhat-developer/odo/pkg/application/labels"
 	componentlabels "github.com/redhat-developer/odo/pkg/component/labels"
 	"github.com/redhat-developer/odo/pkg/occlient"
+	"github.com/redhat-developer/odo/pkg/util"
 
 	log "github.com/sirupsen/logrus"
 )
 
+// urlSourceNameAnnotation indicates the annotation used for the actual URL name (non metadata: name)
+const urlSourceNameAnnotation = "app.kubernetes.io/component-name"
```

Comment:
This constant already exists in https://github.com/redhat-developer/odo/blob/75f494721a31d2113908c6f01fd10f63cd6127cf/pkg/component/labels/labels.go#L8